### ISSUE TITLE
bug: Can't slide ion-range after reorder ion-item

### ISSUE THREAD

**Issue by pauljohnberry** [ionic-team/ionic-framework#19322]
# Bug Report

**Ionic version:**
[x] **4.9.0**

**Current behavior:**
If an <ion-range> is dragged down in <ion-list>, it is possible to slide range item, 
but not possible to slide any of the range items it passed. If the item is initially dragged up in the list, sliding of that range item is no longer possible.

![Screen Recording 2019-09-10 at 09 41 01 mov](https://user-images.githubusercontent.com/32555128/64633109-617f9700-d3af-11e9-9a2d-dcdec9a90b15.gif)

**Expected behavior:**
Sliding should still be possible on all <ion-range> items in the list after reorder.

**Steps to reproduce:**
See gif above

**Related code:**

```
<ion-list>
  <ion-reorder-group (ionItemReorder)="doReorder($event)" disabled="false">
    <ion-item *ngFor="let item of foo">
      <ion-range mode="md" pin="false" color="dark"></ion-range>
      <ion-reorder slot="end">
        <ion-icon name="md-more"></ion-icon>
      </ion-reorder>
    </ion-item>
  </ion-reorder-group>
</ion-list>
```

```
export class HomePage {
  foo = ['a', 'b', 'c', 'd'];

  constructor() {}

  doReorder(ev: any) {
    this.foo = ev.detail.complete(this.foo);
  }
}
```
**Other information:**
Related Issue #17890.
I’ve also tested this issue on web browser (chrome), iOS (iPhone 7plus) and iPhone XR emulator with the same results.

**Ionic info:**

```
Ionic:

   ionic (Ionic CLI)             : 4.10.2 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 4 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/paulberry/Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 7.0.0
   NodeJS            : v10.16.3 (/usr/local/Cellar/node/9.3.0_1/bin/node)
   npm               : 6.9.0
   OS                : macOS Mojave
   Xcode             : Xcode 10.3 Build version 10G8
```

**Comment by lincolnthree** [comment]
FWIW. I have also seen various occurrences of ranges becoming "unclickable" after resizing the browser (using responsive devtool). I hadn't paid much attention to it until seeing this because I figured it was a fluke of dev tools. Will pay a little more attention and see if I can add more info.

**Comment by pauljohnberry** [comment]
Thanks @lincolnthree. Just for extra information, I’ve also tested this issue on web browser (chrome), iOS (iPhone 7plus) and iPhone XR emulator with the same results.

**Comment by DarkLeopard** [comment]
I tried removeEventListener, but it's not help.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Is this still an issue with the latest version of Ionic Framework? If so, please provide a repo with the code required to reproduce this issue.

**Comment by pauljohnberry** [comment]
Hi @liamdebeasi 

Thanks, I've tested and the Problem still seems to exist, here is the repo:

https://github.com/pauljohnberry/ionic-slider-bug/tree/feature/ionicupdate

master contains the original issue, while the feature/ionicupdate has the lastest version as below

Updated as follows:
`Ionic:

   Ionic CLI                     : 6.3.0 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.2.3
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 7.1.4, ios 4.5.5
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 4 other plugins)

Utility:

   cordova-res (update available: 0.15.1) : 0.11.0
   native-run                             : not installed

System:

   ios-deploy : 1.10.0
   NodeJS     : v12.14.0 (/usr/local/bin/node)
   npm        : 6.13.4
   OS         : macOS Catalina
   Xcode      : Xcode 11.5 Build version 11E608c`

**Comment by liamdebeasi** [comment]
Thanks for the issue. This issue was resolved during Ionic v5, but the issue was never closed out. Updating to the latest version of Ionic v5 causes this issue to no longer reproduce, so I am going to close this thread out.

If you encounter any additional problems, please open a new issue and include a code reproduction. Thank you!
